docs/sources/pip.md

```diff
@@ -16,6 +16,7 @@ _note_: `<your_venv_dir>` path should be relative to the repository root or can
 #### virtual_env_dir (Required)
 
 The `pip` command will be sourced from this directory.
+You have to add this setting to `.licensed.yaml`.
```

Comment:
```suggestion
You have to add this setting to your licensed configuration file.
```

Users might be using a different configuration file name

Reply:
That's right. I fix this as it.